Library/Dnn.PersonaBar.UI/admin/personaBar/scripts/permissionGrid.js

```diff
@@ -361,10 +361,16 @@ if (typeof dnn.controls === "undefined" || dnn.controls === null) { dnn.controls
                 return false;
             }
 
+            var defaultPermission = this.data.permissionDefinitions.find(
+                function(permission) {
+                    return permission.permissionId == 5;
+                }
+            );
+            defaultPermission.allowAccess = true;
             this._buildGridRow(this._rolesTable.find('tbody'), {
                 roleId: roleId,
                 roleName: roleName,
-                permissions: []
+                permissions: [defaultPermission]

```

Comment:
Why is it this particular permission though?

Reply:
This is what we need to have the permission checked for "View Folder" as required.